public/js/views/management/pay-methods.jsx

```diff
@@ -35,6 +40,12 @@ export default class PayMethods extends Component {
   }
 
   render() {
+
+    var isDeleteDisabled = !this.props.payMethods ||
```

Comment:
is it ok that this line doesn't end in a semi colon? Do we not need to put parentheses around the expression? Maybe babel fixes it all? ;)